### PR TITLE
Created configuration for DefaultDecisionEngine. Resolved #61

### LINKED ISSUE #61: Create configuration for DefaultDecisionEngine.

### PR DESCRIPTION
**Pull request by mariuszkumor**

### CODE REVIEW

File: src/test/groovy/org/powerflows/dmn/engine/configuration/DefaultDecisionEngineConfigurationSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+import org.powerflows.dmn.engine.model.evaluation.result.DecisionResult
+import org.powerflows.dmn.engine.model.evaluation.result.EntryResult
+import org.powerflows.dmn.io.yaml.YamlDecisionReader
+import spock.lang.Specification
+
+class DefaultDecisionEngineConfigurationSpec extends Specification {
+
+    void 'should configure default decision engine'() {
+        given:
+        final DecisionEngineConfiguration decisionEngineConfiguration = new DefaultDecisionEngineConfiguration()
```
Comment (mariuszkumor) [inline src/test/groovy/org/powerflows/dmn/engine/configuration/DefaultDecisionEngineConfigurationSpec.groovy]: Maybe I'll add a builder for that.

File: src/main/java/org/powerflows/dmn/engine/configuration/DecisionEngineConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+package org.powerflows.dmn.engine.configuration;
+
+
+import org.powerflows.dmn.engine.DecisionEngine;
+import org.powerflows.dmn.engine.reader.DecisionReader;
+
+public interface DecisionEngineConfiguration {
+
+    void setDecisionReader(DecisionReader decisionReader);
```
Comment (stlhrt) [inline src/main/java/org/powerflows/dmn/engine/configuration/DecisionEngineConfiguration.java]: It shouldn't use reader directly, there should be something like decision provider or source that would provide Decisions...

File: src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+id: some_table_id_1
+name: Some Table Name
+hit-policy: UNIQUE
+fields:
+  in:
+    inputOne:
+      description: Some Input 1 Description
+      type: INTEGER
+      expression-type: LITERAL
```
Comment (stlhrt) [inline src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml]: this should be redundant

File: src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      description: Some Output 1 Description
+      type: BOOLEAN
+    outputTwo:
+      description: Some Output 2 Description
+      type: STRING
+rules:
+- description: Some Rule 1 Description
+  in:
+    inputOne:
+      expression-type: LITERAL
```
Comment (stlhrt) [inline src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml]: literal declarations are redundant, use name: value notation

Comment (mariuszkumor) [reply src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml]: intentional, I wanted to have it described in this way

File: src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml (hunk 3 above)
Comment (mariuszkumor) [reply src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml]: intentional, I wanted to have it described in this way

**Pull request merged**